epochStart/bootstrap/interface.go

```diff
@@ -60,3 +59,16 @@ type NodeTypeProviderHandler interface {
 	GetType() core.NodeType
 	IsInterfaceNil() bool
 }
+
+// EpochStartBootstrapperFactoryHandler defines the epoch start bootstrapper factory handler
+type EpochStartBootstrapperFactoryHandler interface {
```

Comment:
suggestion to rename to `EpochStartBootstrapperCreator`

Reply:
renamed